index.rst

```diff
@@ -6,24 +6,219 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
-
-   **This technote is a work-in-progress.**
-
 Abstract
 ========
 
-This is a technote for the description of results from Individual Hardpoint Breakaway Test. 
+This technote describes the analysis and results derived from individual hard point breakaway tests conducted with the M1M3 surrogate.
+The notebook is located within `notebooks_vandv <https://github.com/lsst-sitcom/notebooks_vandv/blob/tickets/SITCOM-838/notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-838_Anaysis.ipynb>`_ GitHub repository.
+With all the steps guarded with reasonable timeouts, so problems are detected if hard point cannot travel to reach low or high limit switches, etc.
+**If this test shows that the hard points do not work properly at the limits, this could be one of the blockers for the installation of M1M3 until it is solved.** 
+
+
+Hardpoint Breakaway Test
+========================
+
+The active support system of the M1M3 includes six axial hard point actutators in the hexapod configuration. :cite:`2018SPIE10700E..3GD`
+These hard point actuators should minimize forces during slews at any TMA position and be kept under the breakaway limit. 
+The breakaway limits for each hard points should happen in the range of -4420 to -3456 N for retraction and 2981 to 3959 N for extension.
+The followings steps are performed during the individual hard point breakaway test.
+
+1. Move hard point in negative (increasing tension) direction until either a low limit switch is actuated
+
+2. Move hard point in positive (increasing compression) direction till the high limit switch is triggered
+
+3. Move hard point downwards (increasing tension) until the low limit switch is hit
+
+4. Move hard point back to reference position
+
+5. Wait for the hard point to reach the reference position
+
+
+Note that for tests only full travel, from low limit to high limit and back, performed in steps 2 and 3 shall be considered. Other hard point movements do not guarantee a full hard point motion range will be examined.
```

Comment:
I understand what you are saying, but I would reword this for a clearer message.

Reply:
I think "Note that" parts can be removed for now. I guess those sentences are written for my reference by Petr.